File: Examples.VB/Features/Charts/Axes/ConfigTickMarkLabelAngle.vb
Namespace Features.Charts.Axes
    Public Class ConfigTickMarkLabelAngle
        Inherits ExampleBase
		Public Overrides Sub Execute(ByVal workbook As GrapeCity.Documents.Excel.Workbook)
            'Open an excel file
            Dim fileStream = GetResourceStream("xlsx\Manufacturing output chart.xlsx")
            workbook.Open(fileStream)

            Dim worksheet As IWorksheet = workbook.Worksheets(0)

            Dim shape As IShape = worksheet.Shapes(0)

            Dim category_axis As IAxis = shape.Chart.Axes.Item(AxisType.Category)

            'config tick label's angle
            category_axis.TickLabels.Orientation = -45
        End Sub

        Public Overrides ReadOnly Property UsedResources() As String()
            Get
                Return New String() {"xlsx\Manufacturing output chart.xlsx"}
            End Get
        End Property
	End Class
End Namespace

File: Examples/Features/Charts/Axes/ConfigTickMarkLabelAngle.cs
using GrapeCity.Documents.Excel.Drawing;
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Charts.Axes
{
    public class ConfigTickMarkLabelAngle : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            //Open an excel file
            var fileStream = this.GetResourceStream("xlsx\\Manufacturing output chart.xlsx");
            workbook.Open(fileStream);

            IWorksheet worksheet = workbook.Worksheets[0];

            IShape shape = worksheet.Shapes[0];
            
            IAxis category_axis = shape.Chart.Axes.Item(AxisType.Category);

            //config tick label's angle
            category_axis.TickLabels.Orientation = -45;
        }

        public override bool IsNew
        {
            get
            {
                return true;
            }
        }

        public override string[] UsedResources
        {
            get
            {
                return new string[] { "xlsx\\Manufacturing output chart.xlsx" };
            }
        }
    }
}
